### DISCUSSION TITLE
Introducing the Signature component ✍️

### DISCUSSION THREAD

**Discussion by jamie-budibase** [Budibase/budibase#13748] [Announcements]
![Signature Hero](https://github.com/Budibase/budibase/assets/154342393/bd553d7f-a9d8-48ce-9017-99907cf2652a)

Users can draw and submit their signatures on Budibase forms with the new signature component. Click "add signature" on either the grid or a form field, and a modal with the signature pad will be added. Once a user submits a signature, it is securely stored as an image in that row. 

[Read the launch blog to learn more.](https://budibase.com/blog/updates/2024/signature-component/)

**Comment by mikesealey** [comment]
Documentation for this feature is now live, and can be found [here](https://docs.budibase.com/docs/signature-field).

**Reply by JCV67** [reply]
Hello,
Is there any way of using signature component with Postgres, tried with a bytea column with no luck.
Thank you

**Reply by ysguy** [reply]
Struggling with the exact same question myself, i'd like the signature to store in the external DB (In my case MySql). it appears that in couchDB or the native BB DB it just stores a text URL and the file is written to the filesystem that is accessed via the URL. In this case a TEXT/VARCHAR ***should*** easily hold the data for the signature. I'm also thinking of creating an automation that will take this URL from the internal DB and copy it to the external DB in another column, awkward but could work - i am very new to Budibase and i haven't figured out automations yet.
